covid_19/covid19_and_era5_matching.ipynb

```diff
@@ -0,0 +1,7632 @@
+{
```

Comment:
<p>I've also seen this calculated over just the time to go from 10-100 or 10-1000 cases. This helps knock down some of the noise in the early days and cuts down on the spurious (for our study) impacts of social distancing. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/Pangeo_KONQUER/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Yeah we can change the growth rate calculation. It was easiest to use np.nonzero to find the index of first case, but I could use 10 as the initial number of cases instead (if we want to use the same formula).